### PR TITLE
DOMA-1891: werf.yaml: add base image and optimise yarn install

### PR DESCRIPTION
**Pull request by pahaz**

### CODE REVIEW

**Comment by pahaz** [comment]
Already deployed to develop!

**Comment by pahaz** [comment]
Before:
Screenshot 2021-12-24 at 19 44 12
After (with cache):
Screenshot 2021-12-24 at 23 46 43

**Review by sitozzz — APPROVED** [review]

**Review by akarjakin — APPROVED** [review]

File: werf.yaml [hunk 1]
```diff
@@ -37,114 +21,131 @@ shell:
...
+  install:
+  - "cd /app"
+  - "yarn install"
   setup:
   - "cd /app"
   - "echo '# Build time .env config!' >> /app/.env"
   - "echo 'COOKIE_SECRET=undefined' >> /app/.env"
   - "echo 'DATABASE_URL=undefined' >> /app/.env"
   - "echo 'WORKER_REDIS_URL=undefined' >> /app/.env"
   - "echo 'NODE_ENV=production' >> /app/.env"
```
Comment (vairell) [inline werf.yaml:63]: Такое же предложение

File: werf.yaml [hunk 2]
```diff
@@ -37,114 +21,131 @@ shell:
...
   - "yarn workspace @app/condo build"
-  - "yarn cache clean"
   - "rm -rf /app/.env"
   - "rm -rf /app/.config"
   - "rm -rf /app/.yarn"
-  - "rm -rf /app/.cache"
 
 docker:
   ENV:
     LANG: C.UTF-8
```
Comment (vairell) [inline werf.yaml:73]: Такое же предложение

File: werf.yaml (hunk 1 above)
Comment (vairell) [reply werf.yaml:63]: https://werf.io/documentation/v1.2/reference/werf_yaml_template_engine.html#example-how-to-use-template-files

File: werf.yaml [hunk 3]
```diff
@@ -37,114 +21,131 @@ shell:
...
+    - "apps/**"
+    - "packages/**"
+    - "bin/**/*"
+  excludePaths:
+  - .github
+  - .helm
+  - werf.yaml
+  - werf-giterminism.yaml
+  owner: app
+  group: app
```
[the remark this answers is not in the document]
Comment (pahaz) [reply werf.yaml:51]: I think it's for the next PR

**Pull request merged**